runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -418,13 +418,13 @@ public void onChatMessage(ChatMessage event)
 		}
 
 		Matcher mProgress = CHAT_GEM_PROGRESS_MESSAGE.matcher(chatMsg);
-
 		if (!mProgress.find())
 		{
 			return;
 		}
 		String taskName = mProgress.group(1);
 		int amount = Integer.parseInt(mProgress.group(2));
+
```

Comment:
Why is this whitespace changed?

Reply:
That's how it was before the other PR messed it up: https://github.com/runelite/runelite/commit/2adb760f02a73e2e0e8bbd600fb60e2928be6ae7#diff-1263f0b025e8719696df37a6081772b9